views/admin_common_get.js

```diff
@@ -5,7 +5,6 @@ const {get_reports} = require("../apis/admin/commons");
 const moment = require('moment');
 
 module.exports = async (req, res) => {
-    const users = await get_users_in_common(req, req.params.commonId);
```

Comment:
Did you find another way to get the user info without this line?

Reply:
I took it out because we are no longer displaying the user information on the `templates/admin_commons.ejs` page; that query was causing the page to timeout, and the information is the same information that's on the report.   

The report takes a long time to generate (17 minutes and 46 seconds to be precise).

Reply:
I took it out because we are no longer displaying the user information on the templates/admin_commons.ejs page; that query was causing the page to timeout, and the information is the same information that's on the report.

The report takes a long time to generate (17 minutes and 46 seconds to be precise).